chutney/action-impl/src/test/java/com/chutneytesting/action/kafka/KafkaBasicConsumeActionTest.java

```diff
@@ -198,6 +199,31 @@ public void should_consume_simple_text_message() {
         assertThat(logger.errors).isEmpty();
     }
 
+    @Test
+    public void should_consume_simple_text_message_with_multiple_values_header() {
```

Comment:
should_consume_simple_text_message_with_multiple_values_f**or_the_same**_header

Reply:
Yes, that's why i didn't put an s to header. If it's confusing, i'll do the change.